docs/contribute/contribute.md

```diff
@@ -194,6 +195,9 @@ Reminders are generated and sent using an Artisan command `monica:sendnotificati
 Monica calculates every night (ie once per day) a set of metrics to help you understand how the instance is being used by users. That will also allow to measure growth over time.
 
 Statistics are generated by the Artisan command `monica:calculatestatistics` every night at midnight and this cron is defined in `app/console/Kernel.php`.
+## Routing
```

Comment:
Please add a line above for better readability.

Also, the text is poorly written.

Please change it to

```
When adding new routes that inject Models, you need to either 
* use Laravel's route helper to inject the model (this is the preferred way)
* if for some reason you can't call the route helper instead of putting the ID in the URL, call `hashID()` on the model you are trying to inject this is to ensure that URLs are not leaking information. 

Also, if this is the first time that Model is being used in a route, ensure that it uses the `hasher` trait.
```

Reply:
Done